### DISCUSSION TITLE
(WiinUSoft) Motion Plus for Gyro Mouse

### DISCUSSION THREAD

**Discussion by Stoutscientist** [KeyPuncher/WiinUPro#146] [Ideas]
Hi! I was wondering if motion plus support was going to be added so that I could control right stick with gyro? I got up and down working but the base Wiimote doesn't have horizontal gyro.

**Comment by KeyPuncher** [comment]
Maybe, I'm planning to get gyro support into WiinUPro 0.9.9 that would then allow it to be opened up to be ported to WiinUSoft.

**Reply by Stoutscientist** [reply]
Awesome! WiinUPro should work too I think, and just out of curiosity, is there an estimated time frame? No rush at all.

**Reply by KeyPuncher** [reply]
Not exactly. I have some vacation time coming up for the second half of December where I might be able to finish the IR stuff I started then re-started for version 0.9.8. If I'm lucky I might be able to get some basic and likely buggy gyro stuff out. If not it will be a while, next closest timeframe would be in the Summer time since I have a fairly intense programming course coming up for Spring.
